src/range_map.rs

```diff
@@ -62,22 +63,23 @@ impl<T> RangeMap<T> {
         }
     }
 
-    /// Provide read-only iteration over everything in the given range.  This does
-    /// *not* split items if they overlap with the edges.  Do not use this to mutate
+    /// Provides read-only iteration over everything in the given range. This does
+    /// **not** split items if they overlap with the edges. Do not use this to mutate
```

Comment:
Why move from italics to bold here? Italics seems like the more appropriate kind of emphasis, IMO.